{{cookiecutter.project_slug}}/Makefile

```diff
@@ -30,7 +30,7 @@ FILEBROWSER_NAME?=filebrowser-{{cookiecutter.project_slug}}
 
 BASE_ENV_NAME?=neuromation/base
 CUSTOM_ENV_NAME?=image:neuromation-{{cookiecutter.project_slug}}
-TRAINING_MACHINE_TYPE?=gpu-small
+TRAINING_MACHINE_TYPE?=cpu-small
```

Comment:
+1. 
Our users will hardly think about the machine type when getting started with a platform. If they begin with CPU, the experience may be somewhat unpleasant (because of speed).

Reply:
Ooops... Sorry, again debugging fix. In one of other PRs we set `TRAINING_MACHINE_TYPE=cpu-small` for `make test_e2e_dev` only, but it's not yet merged.